src/rust/node-identifier/src/lib.rs

```diff
@@ -542,7 +542,7 @@ async fn attribute_asset_ids(
     let mut output_graph = Graph::new(unid_graph.timestamp);
     output_graph.edges = unid_graph.edges;
 
-    let node_asset_ids: HashMap<String, String> = HashMap::new();
+    let _node_asset_ids: HashMap<String, String> = HashMap::new();
```

Comment:
tbh with cases like this where the value is unused I'd rather it be deleted, or warn, rather than hide the warning like this.

Reply:
Yea, I'm going through node identifier currently and the majority of changes there will occur in the #306 PR